compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -511,16 +557,14 @@ execIde telemetry (Debug debug) enableScenarioService options =
                       whenJust gcpStateM $ \gcpState -> Logger.GCP.logIgnored gcpState
                       f loggerH
                   TelemetryDisabled -> f loggerH
-          dlintDataDir <- locateRunfiles $ mainWorkspace </> "compiler/damlc/daml-ide-core"
           options <- pure options
               { optScenarioService = enableScenarioService
               , optEnableOfInterestRule = True
               , optSkipScenarioValidation = SkipScenarioValidation True
               -- TODO(MH): The `optionsParser` does not provide a way to skip
               -- individual options. As a stopgap we ignore the argument to
-              -- --jobs and the dlint config.
+              -- --jobs.
               , optThreads = 0
-              , optDlintUsage = DlintEnabled dlintDataDir True
```

Comment:
Now that `cmdIde` accepts all linter flags, we don't need to hardcode them here.